src/Development/IDE/Core/Compile.hs

```diff
@@ -232,7 +232,7 @@ getModSummaryFromBuffer fp contents dflags parsed = do
           { ml_hs_file  = Just fp
           , ml_hi_file  = derivedFile "hi"
           , ml_obj_file = derivedFile "o"
-#ifndef GHC_STABLE
+#if __GLASGOW_HASKELL__ >= 808
```

Comment:
What we really want is to test against the GHC API version. It probably make sense to add a Cabal flag that switches the dependency on `ghc` to a dependency on `ghc-lib` and then we either check on the version of GHC (sidenote: `MIN_VERSION_ghc` seems slightly more appropriate than `__GLASGOW_HASKELL__` but ofc it doesn’t make a difference) or `MIN_VERSION_ghc_lib`. I’ll see if I can tackle this as part of #64 so we can get some automated testing before this (ofc if anybody else wants to give it a shot your more than welcome to, just let me know so we avoid duplicating work).

Reply:
I’ve just merged #73 which adds a `MIN_GHC_API_VERSION` macro that should do the right thing™. It would be great if you could rebase this PR and update it to include this. Let me know if you run into any issues and I’m happy to help.